src/main/java/org/opensearch/security/dlic/rest/api/AccountApiAction.java

```diff
@@ -62,7 +62,7 @@
  */
 public class AccountApiAction extends AbstractApiAction {
     private static final String RESOURCE_NAME = "account";
-    private static final List<Route> routes = addRoutesPrefix(ImmutableList.of(
+    private static final List<ReplacedRoute> replacedRoutes = addRoutesPrefix(ImmutableList.of(
             new Route(Method.GET, "/account"),
```

Comment:
Is `/account` being deprecated here? Shouldn't it be `_opendistro/_security/api/account`

Reply:
I am not sure, but even with `routes()`, we were supporting both `_plugins/` and `_opendistro`.
With `replaceRoute()` we are still supporting both `_plugins` and `_opendistro`, however we are sending a warning message to the user for `_opendistro`.
Do you mean `/account` should only work with `_opendistro`?

Reply:
NVM. I found the answer.